apps/axiom/src/app/(payload)/_public/login/page.tsx

```
@@ -0,0 +1,86 @@
+  },
+  {
+    name: "Discord",
+    url: "/api/users/oauth/discord",
+    icon: "https://cdn.prod.website-files.com/6257adef93867e50d84d30e2/636e0a69f118df70ad7828d4_icon_clyde_blurple_RGB.svg",
+    bgColor: "#E0E3FF",
+  },
+];
+
+const OAuthButton = ({ href, icon, bgColor, name }) => (
+  <Link href={href} style={{ width: '100%' }}>
+    <button style={{ ...buttonStyles, backgroundColor: bgColor }}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)